### PR TITLE
Implement transactions' add/drop player APIs

### PR DESCRIPTION
**Pull request by githubsmilo**
issue #6

* add player
```
transactions.add_player(
    league_key,
    team_key,
    player_key,
    function(err, data) {
      if (err)
        // handle error. do your thing
    });
```
* drop player
```
transactions.drop_player(
    league_key,
    team_key,
    player_key,
    function(err, data) {
      if (err)
        // handle error. do your thing
    });
```
* add and drop players
```
transactions.adddrop_players(
    league_key,
    team_key,
    player_key, // add
    player_key, // drop
    function(err, data) {
      if (err)
        // handle error. do your thing
    });
```

### CODE REVIEW

**Comment by whatadewitt** [comment]
SNAP! I just saw this. I will merge in tonight.

File: collections/transactionsCollection.js [hunk 1]
```diff
@@ -78,3 +78,93 @@ TransactionsCollection.prototype.leagueFetch = function(leagueKeys, resources, f
 };
 
 // todo: http://fantasysports.yahooapis.com/fantasy/v2/league/{league_key}/transactions;types=waiver,pending_trade;team_key={team_key}
+
+TransactionsCollection.prototype.add_player = function(leagueKey, teamKey, playerKey, cb) {
+  var url = 'http://fantasysports.yahooapis.com/fantasy/v2/league/' + leagueKey + '/transactions?format=json';
+  var xmlData = ' \
```
Comment (whatadewitt) [inline collections/transactionsCollection.js]: WOW! do we really need to convert stuff to XML??

File: index.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ function YahooFantasy(consumerKey, consumerSecret) {
     team: teamResource(),
     teams: teamsCollection(),
     transaction: transactionResource(),
-    // transactions: transactionsCollection(),
+    transactions: transactionsCollection(),
```
Comment (whatadewitt) [inline index.js]: I've been doing a lot of refactoring... there's a good chance I'll have an update by the weekend... would change a lot of this :S

**Comment by whatadewitt** [comment]
as i mentioned in a comment. I'm going through a pretty serious refactor right now to simplify some stuff and make the code better for unit testing. Can you only commit the add/drop stuff for now, just to keep the size of the PR down? there's a lot more going on than just add/drop players.

File: collections/transactionsCollection.js (hunk 1 above)
Comment (githubsmilo) [inline collections/transactionsCollection.js]: Yes, we need 'POST' request with some XML data to add/drop players. :smiley_cat: 
Refer this site, https://developer.yahoo.com/fantasysports/guide/#post

**Comment by githubsmilo** [comment]
Cool! I will wait for your refactoring works. 
If you want a new pull request of add/drop API after refactoring, I will do it.

**Comment by whatadewitt** [comment]
Ok, thanks!

File: collections/transactionsCollection.js (hunk 1 above)
Comment (whatadewitt) [inline collections/transactionsCollection.js]: Gross!

**Comment by githubsmilo** [comment]
I checked your refactoring works! Great! I will create a new pull request of add/drop API tonight. (This pull request is closed)

**Pull request closed without merging**